<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prog_Calc</class>
 <widget class="QMainWindow" name="Prog_Calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prog_Calc</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="Edit_Calc_Num">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>350</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Hex_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>350</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="A_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="L_Shift_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="R_Shift_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cancel_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Space_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Dec_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>350</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>DEC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Oct_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>350</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>OCT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bin_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>350</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>BIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="C_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="D_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="E_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Paren_Right_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Percent_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>％</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Paren_Left_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Division_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Num_Eight_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Num_Nine_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Num_Seven_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Multiply_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Num_Five_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Num_Six_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Num_Four_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Subtract_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Num_Two_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Num_Three_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Num_One_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Num_Zero_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Dec_Point_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Reversal_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>±</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Equal_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Edit_Result_Num">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>350</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Prog_Calc.qrc"/>
 </resources>
 <connections/>
</ui>
